packages/cli/generation/remote-generation/remote-workspace-runner/src/runRemoteGenerationForGenerator.ts

```diff
@@ -161,3 +177,24 @@ async function computeSemanticVersion({
     }
     return response.body.version;
 }
+
+async function uploadSources({
+    context,
+    sourceUploader,
+    sourceUploads
+}: {
+    context: InteractiveTaskContext;
+    sourceUploader: SourceUploader;
+    sourceUploads: Record<FdrAPI.api.v1.register.SourceId, FdrAPI.api.v1.register.SourceUpload>;
+}): Promise<SourceConfig> {
+    if (sourceUploads == null && sourceUploader.getSourceTypes().has("protobuf")) {
```

Comment:
can `sourceUploads` even be null here? based on the type signature looks like not?

Reply:
Great catch - this was from before a refactor. Will update 👍